tetratelabs/istio-security-analyzer
# Detect gateways for relaxed SNI host matching and report errors.

Steps to test:
1. Git pull virtual-service-check branch.
2. Take a back up before removing /pkg/parser/testdata/vs-deny-relaxed-sni.yaml file.
3. Run it, security issue will be there in the report, saying which gateway having what host(s) which can create problem in other gateway configuration.
4. Paste the file at its original place, which was deleted in step no. 2 and run it again.
5. It means a virtual service has already configured to handle that relaxed sni host.
